src/widgets/Tabs/PatientEdit.js

```diff
@@ -79,7 +82,15 @@ const PatientEditComponent = ({
         <View style={localStyles.verticalSeparator} />
         <View style={localStyles.formContainer}>
           {surveySchema && (
-            <JSONForm ref={formRef} onSubmit={onSubmitSurvey} surveySchema={surveySchema}>
+            <JSONForm
+              onChange={data => {
+                updateForm(data.formData, data.errors);
+              }}
```

Comment:
passing the formdata and any errors (because live validation is enabled) up to the store to enable/disable the next button. Figured that's easier than checking when clicking next and warning, even though I kinda prefer on submit validation